doc/tutorial/statistical_inference/supervised_learning.rst

```diff
@@ -374,7 +374,9 @@ function or **logistic** function:
 
 ::
 
-    >>> logistic = linear_model.LogisticRegression(C=1e5)
+    >>> logistic = linear_model.LogisticRegression(C=1e5, solver='liblinear',
+    ...                                            fit_intercept=True, 
```

Comment:
Hmm... I think changes like this shouldn't be necessary if we merge #11476 first